test/check-existing-system

```diff
@@ -153,6 +166,12 @@ class TestFedoraPlansUseFreeSpace(VirtInstallMachineCase):
         self.install(needs_confirmation=True)
         self.verifyDualBoot(root_one_size=14.2, root_two_size=4.7)
 
+class TestFedoraPlansUseFreeSpaceBIOS(_TestFedoraPlansUseFreeSpace):
+    efi = False
```

Comment:
If this should not be something like `self.efi`?

Reply:
Not sure what you ask

Reply:
Oh I guess you mean that I can instead setup it in the constructor with self.efi = False. Sure.